OpenRA.Game/Traits/ActivityUtils.cs

```diff
@@ -12,6 +12,7 @@
 using System.Diagnostics;
 using OpenRA.Activities;
 using OpenRA.Support;
+using Activity = OpenRA.Activities.Activity;
```

Comment:
Is this needed?

Reply:
'Activity' is an ambiguous reference between 'OpenRA.Activities.Activity' and 'System.Diagnostics.Activity'

Reply:
Ah, these seemingly unrelated changes related to the switch from `netstandard` to `net5` then? Might be cleaner to pull these plus the tfm changes to their own PR?

Reply:
Please do pull these changes out into their own commit (e.g. "Compile engine and mod dlls as NET5") even if we don't want to split out a new PR. This will make things easier to understand when looking back at the git history/blame in the future.